doc/Makefile

```diff
@@ -54,6 +58,12 @@ html:
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
+
+netlify:
+	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) -c source/conf-netlify $(BUILDDIR)/html
+	@echo
+	@echo "Netlify build finished. The HTML pages are in $(BUILDDIR)/html."
+	
```

Comment:
I guess that part can be removed and replaced with one for render.com instead, @patrickbirch

Reply:
Sure.